### PR TITLE
55 fix python make

### LINKED ISSUE #55: All python notes fail on `create_version()`
There is currently no logic to handle python notes in `create_version()`

This causes all python notes builds to fail.

### PR DESCRIPTION
**Pull request by jwalton3141**
Fixes #55 

All python builds currently fail from `get_logo_path.R`'s `create_version()`.

### CODE REVIEW

**Comment by csgillespie** [comment]
This is  a bit hellish of a merge. Can you just pull out the changes and create a clean PR?

**Comment by jwalton3141** [comment]
@csgillespie no I don't wanna. Merged and "files changed" looks correc, so should be reviewable...

File: R/get_logo_path.R [hunk 1]
```diff
@@ -93,7 +93,10 @@ create_version = function() {
     pkg_name = get_r_pkg_name()
     pkg_ver = packageVersion(pkg_name)
   } else {
-    pkg_ver = ""
+    pkg_name = get_python_pkg_name()
+    pkg_ver = system2("pip", c("show", pkg_name, "|",
```
Comment (csgillespie) [inline R/get_logo_path.R:97]: Should that be `pip3` to be clear? Also, this is in a venv, will this still work? I presume yes

**Comment by csgillespie** [comment]
> @csgillespie no I don't wanna. Merged and "files changed" looks correc, so should be reviewable...

You need to make sure you squash then, otherwise the commit history will be screwed.

**Pull request closed without merging**